Formula/qtkeychain.rb

```diff
@@ -18,7 +18,6 @@ class Qtkeychain < Formula
   depends_on "qt@5"
```

Comment:
Do we want to try switching to Qt6? Would require building with `BUILD_WITH_QT6=ON` and updating test.

Upstream claims it is supported since QtKeychain 0.12 and it doesn't seem to be part of any dependency tree that will cause issues. Not sure if there was any reason why we kept `qt@5`.

Reply:
I will push a commit to change this.  I'm guessing the continued use of `qt@5` was just an oversight.